raven/utilities/geoserver.py

```diff
@@ -46,11 +45,15 @@
     Intersects = None
     wfs_Point = None
 
+from .geo import determine_upstream_ids
+
 # Do not remove the trailing / otherwise `urljoin` will remove the geoserver path.
-# Can be set at runtime with `$ env GEO_URL=https://xx.yy.zz/geoserver/ ...`.
-GEO_URL = os.getenv("GEO_URL", "https://pavics.ouranos.ca/geoserver/")
+# Can be set at runtime with `$ env RAVENPY_GEOSERVER_URL=https://xx.yy.zz/geoserver/ ...`.
+GEOSERVER_URL = os.getenv(
+    "RAVENPY_GEOSERVER_URL", "https://pavics.ouranos.ca/geoserver/"
+)
 
```

Comment:
One way to keep this backward compat is to keep `GEO_URL = os.getenv("GEO_URL", "https://pavics.ouranos.ca/geoserver/")` and change `GEOSERVER_URL` to `GEOSERVER_URL = os.getenv("RAVENPY_GEOSERVER_URL", GEO_URL)`

Reply:
Done.